tests/test_config_consistency.py

```diff
@@ -662,13 +661,34 @@ def get_runtime_metrics(agent):
     return runtime_metrics_gauges, runtime_metrics_sketches
 
 
-# Parse the JSON-formatted log message from stdout and return the 'dd' object
+# Parses the JSON-formatted log message from stdout and returns it
 def parse_log_injection_message(log_message):
     for data in stdout.get_data():
         try:
             message = json.loads(data.get("message"))
         except json.JSONDecodeError:
             continue
+        # For Nodejs
         if message.get("dd") and message.get(log_injection_fields[context.library.library]["message"]) == log_message:
```

Comment:
Can we simplify this to:
```suggestion
        if context.library == "nodejs":
```